big_scape/network/network.py

```diff
@@ -74,41 +84,77 @@ def get_connected_components(
             and_(
                 distance_table.c.record_a_id.in_(
                     select(DB.metadata.tables["connected_component"].c.record_id).where(
-                        DB.metadata.tables["connected_component"].c.id == cc_id
+                        DB.metadata.tables["connected_component"].c.id == cc_id,
+                        DB.metadata.tables["connected_component"].c.cutoff == cutoff,
```

Comment:
I think this is the fix I applied in the other pull request as well, so that PR may not be necessary if we merge this